src/braket/experimental/autoqasm/program/program.py

```diff
@@ -84,6 +89,22 @@ def __init__(self, oqpy_program: oqpy.Program, has_pulse_control: bool = False):
         self._oqpy_program = oqpy_program
         self._has_pulse_control = has_pulse_control
 
+    def bind_calibrations(self, gate_calibrations: Union[Callable, List[Callable]]) -> Program:
+        """Binds the gate calibrations to the program.
+
+        Args:
+            gate_calibrations (Union[Callable, List[Callable]]): The gate calibrations to bind.
+        """
+        if isinstance(gate_calibrations, Callable):
+            gate_calibrations = [gate_calibrations]
+        assert all(isinstance(gc, Callable) for gc in gate_calibrations)
+
+        combined_oqpy_program = oqpy.Program()
+        for gc in gate_calibrations:
+            combined_oqpy_program += gc().oqpy_program
+        self._oqpy_program = combined_oqpy_program + self._oqpy_program
```

Comment:
Is there a reason to do it this way, instead of just:
```
for gc in gate_calibrations:
    self._oqpy_program += gc().oqpy_program
```

Reply:
In that way the defcals would be added to the last. It's looks nicer when decals are on top.